### PR TITLE
Upgrade all dependencies to the latest major versions

### PR DESCRIPTION
**Pull request by erayb**
As part of the major upgrade, as advised by the Origami team, this also deletes previously defined use cases. I have checked the depended projects and they don't seem to be using them anyway.

It will also fix a small UI issue where background of B2B details wasn't looking correct.
Screen Shot 2019-11-27 at 13 53 38

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		"js": "demos/src/demo.js",
 		"sass": "demos/src/demo.scss",
 		"documentClasses": "",
-		"dependencies": "o-fonts@^3.0.0,o-grid@^4.3.3,o-colors@^4.0.0,o-normalise@^1.0.0"
+		"dependencies": "o-fonts@^4.0.0,o-grid@^5.0.0,o-colors@^5.0.1,o-normalise@^2.0.0"
```
Comment (notlee) [inline origami.json]: we updated the spec so `dependencies` should be an array. It was already a string, but just whilst it's visible 😊 
```suggestion
		"dependencies": [
			"o-fonts@^4.0.0",
			"o-grid@^5.0.0",
			"o-colors@^5.0.1",
			"o-normalise@^2.0.0"
		]
```

File: src/scss/_themes.scss [hunk 2]
```diff
@@ -1,47 +1,56 @@
...
 		padding-right: $o-subs-card-horizontal-padding;
 		padding-left: $o-subs-card-horizontal-padding;
 	}
 
 	.o-subs-card__select-button {
-		@include oButtonsTheme(inverse);
+		@include oButtonsContent($opts: (
+			'type': 'secondary',
+			'size': 'big',
+			'theme': ('inverse')
```
Comment (notlee) [inline src/scss/_themes.scss:23]: As a modifier, I think we can reduce the css bundle size a little here by passing `$include-base-styles: false` as a second argument. Which means we don't need to specify `'size': 'big'` either, as it doesn't need to override the default style

File: src/scss/_themes.scss [hunk 3]
```diff
@@ -1,47 +1,56 @@
...
 
 	.o-subs-card__select-button {
-		@include oButtonsTheme($theme: (
-			background: "white",
-			accent: oColorsGetUseCase(o-subs-card-discount, background),
-			colorizer: 'primary'
+		@include oButtonsContent($opts: (
+			'type': 'primary',
+			'size': 'big',
+			'theme': (
```
Comment (notlee) [inline src/scss/_themes.scss:46]: Same here regarding `$include-base-styles: false`

**Review by notlee — APPROVED** [review]
I commented with some suggestions to reduce the css bundle a little, but this looks great 🎉

File: src/scss/_themes.scss (hunk 2 above)
Comment (erayb) [reply src/scss/_themes.scss:23]: Tried this suggestion but we then realised I actually didn't have the base styles in this.

**Pull request merged**